extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.css

```diff
@@ -31,7 +31,7 @@
  */
 .amp-lightboxed {
   position: fixed !important;
-  z-index: 2147483645 !important;
+  z-index: 2147483644 !important;
```

Comment:
this seems to be higher than that of sidebar? that does not seem right. What happens when you trigger a sidebar from inside a lightbox?